blue_interface/blue_interface.py

```diff
@@ -279,22 +343,22 @@ def gripper_enabled(self):
         """
         return self._gripper_enabled
 
-    def inverse_kinematics(self, position, orientation, solver="trac-ik", seed_joint_positions=[]):
+    def inverse_kinematics(self, position, orientation, seed_joint_positions=[]):
```

Comment:
Is the idea here to keep things simple and abstract the IK? I do agree that this is cleaner and it's probably better to keep it like this until we decide to support other solvers.